src/dmn/trn_transit_xdp_usr.h

```diff
@@ -214,15 +214,15 @@ int trn_remove_prog(struct user_metadata_t *md, unsigned int prog_idx);
 
 int trn_update_transit_network_policy_primary_map(struct user_metadata_t *md,
 						  struct vsip_cidr_t *cidr,
-						  __u64 bitmap);
+						  __u64 *bitmap);
```

Comment:
what is the reason to change this param to type of pointer?

Reply:
I need pass in an array instead of individual values

Reply:
in order to pass an array, usually the counter nneds to be one of the params too.